server/src/app.py

```diff
@@ -75,21 +75,61 @@ def configure_app():
     return json(data)
 
 
+@app.route('/timetoclose-comparison', methods=["POST"])
```

Comment:
Lets create a ticket to adjust our routes after presentation

im thinking something along the lines of `/comparison/timetoclose` that way we have `/timetoclose` and `/comparison/timetoclose`

Reply:
sounds good, I can make it